pkg/cmd/env/rc.go

```diff
@@ -0,0 +1,187 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
```suggestion
SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
```
use the current year for newly introduced files

Reply:
✔️